<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>309</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>FPGA Communicator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="group_serial">
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_refreshPortList">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_disconnect">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="cmb_serialPorts"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbl_current_state">
        <property name="text">
         <string>Not connected</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_state">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_flight">
       <attribute name="title">
        <string>Flight</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="group_data">
          <item>
           <widget class="QLCDNumber" name="lcd_timer"/>
          </item>
          <item>
           <widget class="QTableView" name="tbl_data"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="group_plot">
          <item>
           <widget class="QLabel" name="lbl_flightplot">
            <property name="text">
             <string>Flightplot:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCustomPlot" name="plot_flight" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_flightstate">
            <property name="text">
             <string>Status</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
       <zorder>verticalLayoutWidget</zorder>
       <zorder>verticalLayoutWidget_2</zorder>
      </widget>
      <widget class="QWidget" name="tab_messages">
       <attribute name="title">
        <string>Messages</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_msgin">
          <property name="text">
           <string>Ingoing:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lbl_msgout">
          <property name="text">
           <string>Outgoing:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="tbl_msgin"/>
        </item>
        <item row="1" column="1">
         <widget class="QTableView" name="tbl_msgout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
